### PR TITLE
Native AppStreams documentation

### PR DESCRIPTION
**Pull request by cbbayburt**
# Description

Documentation for native AppStreams support.

# Target branches

* SUMA 5.0 and Uyuni only

# Links

- Fixes https://github.com/SUSE/spacewalk/issues/24091
- EPIC: https://github.com/SUSE/spacewalk/issues/22300

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+- Add AppStream section to client configuration guide
+- Add reference guide for Systems > Software > AppStreams
+- Remove warning about limited AppStream support
```
Comment (keichwa) [inline CHANGELOG.md:3]: I think we again are in favor of past tense here.
```suggestion
- Added AppStream section to client configuration guide
- Added reference guide for Systems > Software > AppStreams
- Removed warning about limited AppStream support
```

File: modules/client-configuration/pages/snippets/managing_appstreams.adoc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.Procedure: Managing enabled AppStream modules
+. In the {productname} {webui}, navigate to menu:Systems[Software > AppStreams].
+. From the table of module streams, select the streams to be enabled on the client.
+. Click btn:[Apply Changes]
```
Comment (keichwa) [inline modules/client-configuration/pages/snippets/managing_appstreams.adoc:4]:
```suggestion
. Click btn:[Apply Changes].
```

**Review by keichwa — APPROVED** [review]
Thanks!  Two cosmetic suggestions.

File: modules/client-configuration/pages/appstreams.adoc [hunk 3]
```diff
@@ -0,0 +1,7 @@
+[[appstreams]]
```
Comment (0rnela) [inline modules/client-configuration/pages/appstreams.adoc:1]:
```suggestion
[[appstream-modules]]
```

**Review by 0rnela — APPROVED** [review]

**Pull request merged**